File: backend/backend/DTO/Chat/MessageDTO.cs
using System;

namespace LifeGuideProject.API.DTO.Chat
{
    public class MessageDTO
    {
        public string senderUserEmail { get; set; }
        public string message { get; set; }
        public string receiverUserEmail { get; set; }
        public string connectionId { get; set; }
        public DateTime timeStamp { get; set; }
        public int messageYear { get; set; }
        public int messageMonth { get; set; }
        public int messageDay { get; set; }
        public int messageHour { get; set; }
        public int messageMinute { get; set; }
    }
}

File: frontend/src/app/Models/MessageDTO.ts
export class MessageDTO {
    public id:number = 0; 
    public senderUserEmail: string = '';
    public message: string = '';
    public timeStamp: Date = new Date();
    public connectionId: string = '';
    public receiverUserEmail: string = '';
    public messageYear: number;
    public messageMonth: number;
    public messageDay: number;
    public messageHour: number;
    public messageMinute: number;

    constructor(id :number, senderUserEmail:string,message :string,timeStamp : Date, connectionId : string, 
      receiverUserEmail : string, messageYear: number, messageMonth: number, messageDay: number, messageHour: number, messageMinute: number){
      this.senderUserEmail = senderUserEmail
      this.receiverUserEmail = receiverUserEmail
      this.id = id;
      this.message = message;
      this.connectionId = connectionId;
      this.timeStamp = timeStamp;
      this.messageYear = messageYear;
      this.messageMonth = messageMonth;
      this.messageDay = messageDay;
      this.messageHour = messageHour;
      this.messageMinute = messageMinute;
  }
  }